doc/rippled-example.cfg

```diff
@@ -987,23 +972,6 @@ pool.ntp.org
 [ips]
 r.ripple.com 51235
 
-# Public keys of the validators that this rippled instance trusts.  The latest
```

Comment:
Replace these removed blocks with a `[validators_file]` block.

For bonus points, include a commented out section with this original info and an explanation like "We recommend including your validators in a separate validators_file for reasons, but if you want to quickly bootstrap a server, uncomment these sections."